app/services/taxonomy/taxonomy_query.rb

```diff
@@ -29,6 +29,12 @@ def content_tagged_to_taxons(content_ids, slice_size: 50)
       content_id_hashes.map { |h| h['content_id'] }.uniq
     end
 
+    def parent(content_id)
```

Comment:
I feel that this functionality should already exist somewhere? A [taxon](https://github.com/alphagov/content-tagger/blob/master/app/models/govuk_taxonomy/taxon.rb) has a `parent_node` attribute, isn't that the same thing?